src/Faker/Provider/Base.php

```diff
@@ -263,7 +263,7 @@ public static function shuffle($arg = '')
      * @param array $array The set to shuffle
      * @return array The shuffled set
      */
-    public static function shuffleArray($array = array())
+    public static function shuffleArray(array $array = array())
```

Comment:
I'll remove this change, if someone can explain why it is an BC; I understand that it's looks like it, but I don't see any situation, where it will be a problem. Cuz this function can't be used without an array as param.